devices/models.py

```diff
@@ -128,7 +128,10 @@ class Device(models.Model):
 
     department = models.ForeignKey("users.Department", null=True, blank=True, related_name="devices", on_delete=models.SET_NULL)
     is_private = models.BooleanField(default=False)
+    used_in = models.ForeignKey('self', null = True, blank = True, on_delete = models.SET_NULL,)
 
+    def return_id (self):
+        return self.id
```

Comment:
what's this?

Reply:
not necessary, now that I think about it XD